tfconfig/load_legacy.go

```diff
@@ -110,6 +111,7 @@ func loadModuleLegacyHCL(fs FS, dir string) (*Module, Diagnostics) {
 					Description: block.Description,
 					Default:     block.Default,
 					Required:    block.Default == nil,
+					Sensitive:   block.Sensitive,
```

Comment:
I don't think we need these changes in the legacy code path. This should only be hit if there are parsing errors due to pre-0.12 configuration language, and since sensitive variables were introduced much later, trying to parse them here isn't helpful.

If I'm mistaken and it's worth keeping these changes here, can you add test coverage? Tests currently still pass after reverting the changes to this file locally.

Reply:
I removed the sensitive property for legacy modules. Rationale seems sound to me!